File: UBSHackathonBackend/src/main/java/com/ubs/RegistrationSubmission.java
package com.ubs;

import lombok.Data;
import lombok.RequiredArgsConstructor;

import java.util.List;

@RequiredArgsConstructor
@Data
public class RegistrationSubmission {
    private final String userName;
    private final List<String> cuisines;
    private final List<String> interests;
    private final String bio;
}

File: frontend/src/app/registrationsubmission.ts
export class RegistrationSubmission {
  constructor(username: string, cuisines: string[], interests: string[], bio: string) {
    this.username = username;
    this.cuisines = cuisines;
    this.interests = interests;
    this.bio = bio;
  }

  username: string;
  cuisines: string[];
  interests: string[];
  bio: string;
}
